modules/layers/src/icon-layer/icon-layer.js

```diff
@@ -55,7 +56,7 @@ const DEFAULT_TEXTURE_MAG_FILTER = GL.LINEAR;
  */
 const defaultProps = {
   iconAtlas: null,
-  iconMapping: {type: 'object', value: {}, async: true},
+  iconMapping: null,
```

Comment:
This breaks the async loading support for URL strings supplied to this prop. Please test examples/website/icon